packages/happy-dom/src/query-selector/QuerySelector.ts

```diff
@@ -205,12 +205,6 @@ export default class QuerySelector {
 			);
 		}
 
```

Comment:
We should cover the case if selector is a Symbol.

The correct error for Symbol should be:
`TypeError: Failed to execute 'querySelector' on 'Document': Cannot convert a Symbol value to a string`

```suggestion
		if (typeof selector === 'symbol') {
			throw new window.TypeError(
				`Failed to execute 'querySelector' on 'Document': Cannot convert a Symbol value to a string`
			);
		}
```

Reply:
That check is already included below:

https://github.com/capricorn86/happy-dom/blob/7a5aa6249e7b1fb61aae6ecb64e2ce9384b7e66b/packages/happy-dom/src/query-selector/QuerySelector.ts#L214-L216

Reply:
I've updated that error message to include the node type, as per the other checks.